README.md

```diff
@@ -121,3 +111,41 @@ largest activity ranges across samples. The second part builds a limma model to
 detect signatures with differential temporal patterns between treatment
 and control.
 
+
+### Installation
+
+You can install the latest development version from github with
+``` r
+install.packages("devtools")
+devtools::install_github("greenelab/ADAGEpath")
+```
+If you want to build the vignettes, run
+``` r
+devtools::install_github("greenelab/ADAGEpath", build_vignettes = TRUE)
+```
+
+#### Potential problems during installation:
+
+- make: gfortran-4.8: No such file or directory.
+
+    `ADAGEpath` depends on the Bioconductor package
+`impute` that requires gfortrain-4.8. Install `gfortrain-4.8` to solve this problem.
+You can follow instructions here to install `gfortran-4.8` on Mac
```

Comment:
I have only tested on Mac and Linux and encountered this error on Mac. I need to test on a windows machine.